initfiles/componentfiles/configxml/espsmcservice.xsd.in

```diff
@@ -49,10 +49,10 @@
                 </xs:annotation>
             </xs:attribute>
                 <xs:annotation>
-	          <xs:appinfo>
-	          <!--DOCS_Autobuild_Code-->
-          	    <docid>SMC-T01</docid>
-	          </xs:appinfo>
+              <xs:appinfo>
```

Comment:
@g-pan @JamesDeFabia please comment on what this element should be and if there are any missing docid elements, etc.